src/app/pages/network/configuration/configuration.component.ts

```diff
@@ -0,0 +1,325 @@
+    fcName: 'nameserver1',
+    label: helptext.nameserver1_placeholder,
+    tooltip: helptext.nameserver1_tooltip,
+  };
+
+  nameserver2 = {
+    fcName: 'nameserver2',
+    label: helptext.nameserver2_placeholder,
+    tooltip: helptext.nameserver2_tooltip,
+  };
+
+  nameserver3 = {
+    fcName: 'nameserver3',
+    label: helptext.nameserver3_placeholder,
+    tooltip: helptext.nameserver3_tooltip,
+  };
+
+  ipv4gateway = {
+    fcName: 'ipv4gateway',
+    label: helptext.ipv4gateway_placeholder,
+    tooltip: helptext.ipv4gateway_tooltip,
+  };
+
+  ipv6gateway = {
+    fcName: 'ipv6gateway',
+    label: helptext.ipv6gateway_placeholder,
+    tooltip: helptext.ipv6gateway_tooltip,
+  };
+
+  outbound_network_activity = {
+    fcName: 'outbound_network_activity',
+    label: '',
+    tooltip: '',
+    options: of([
+      // deny type + empty list
+      {
+        label: helptext.outbound_network_activity.allow.placeholder,
+        value: NetworkActivityType.Deny,
+        tooltip: helptext.outbound_network_activity.allow.tooltip,
+      },
+      // allow type + empty list
+      {
+        label: helptext.outbound_network_activity.deny.placeholder,
+        value: NetworkActivityType.Allow,
+        tooltip: helptext.outbound_network_activity.deny.tooltip,
+      },
+      {
+        label: helptext.outbound_network_activity.specific.placeholder,
+        value: 'SPECIFIC',
+        tooltip: helptext.outbound_network_activity.specific.tooltip,
+      },
+    ]),
+  };
+
+  outbound_network_value = {
+    fcName: 'outbound_network_value',
+    label: '',
+    tooltip: helptext.outbound_network_value.tooltip,
+    options: this.ws.call('network.configuration.activity_choices').pipe(
+      map((choices) => {
+        return choices.map(([value, label]) => ({ label, value }));
+      }),
+    ),
+    hidden: true,
+  };
+
+  httpproxy = {
+    fcName: 'httpproxy',
+    label: helptext.httpproxy_placeholder,
+    tooltip: helptext.httpproxy_tooltip,
+  };
+
+  netwait_enabled = {
+    fcName: 'netwait_enabled',
+    label: helptext.netwait_enabled_placeholder,
+    tooltip: helptext.netwait_enabled_tooltip,
+  };
+
+  netwait_ip = {
+    fcName: 'netwait_ip',
+    label: helptext.netwait_ip_placeholder,
+    tooltip: helptext.netwait_ip_tooltip,
+    hidden: true,
+  };
+
+  hosts = {
+    fcName: 'hosts',
+    label: helptext.hosts_placeholder,
+    tooltip: helptext.hosts_tooltip,
+  };
+
+  constructor(
+    private ws: WebSocketService,
+    private slideInService: IxSlideInService,
+    private errorHandler: FormErrorHandlerService,
+    private cdr: ChangeDetectorRef,
+    private fb: FormBuilder,
+    private dialogService: DialogService,
+  ) {}
+
+  ngOnInit(): void {
+    this.isFormLoading = true;
+    this.loadConfig();
+
+    this.form.controls.outbound_network_activity.valueChanges.pipe(untilDestroyed(this)).subscribe(
+      (value: NetworkActivityType) => {
+        if (
+          value === NetworkActivityType.Allow
```

Comment:
`[NetworkActivityType.Allow, NetworkActivityType.Deny].includes(value)` for a single line